core/arch/arm/mm/sub.mk

```diff
@@ -9,7 +9,11 @@ endif
 srcs-y += tee_mm.c
 srcs-y += pgt_cache.c
 srcs-y += mobj.c
+ifeq ($(CFG_CORE_SPCI),y)
+srcs-y += mobj_ffa.c
```

Comment:
Can we make this `srcs-$(CFG_CORE_FFA) += mobj_ffa.c` with no `ifeq`?
It is always a bit suspect when the name of the file does not reflect the config name.

Reply:
Sorry, a leftover that got overwritten in later commits, I'll fix.